<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Phaedros" name="net.groot.data.TestNgRegisterData" tests="4" failures="1" timestamp="25 Feb 2020 03:02:47 GMT" time="6.709" errors="0">
  <testcase name="findRegisterPage" time="0.817" classname="net.groot.data.TestNgRegisterData"/>
  <testcase name="confirm" time="5.211" classname="net.groot.data.TestNgRegisterData"/>
  <testcase name="registerPage" time="0.652" classname="net.groot.data.TestNgRegisterData">
    <failure type="org.testng.TestNGException" message="
The data provider is trying to pass 2 parameters but the method net.groot.data.TestNgRegisterData#registerPage takes 4 and TestNG is unable in inject a suitable object">
      <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 2 parameters but the method net.groot.data.TestNgRegisterData#registerPage takes 4 and TestNG is unable in inject a suitable object
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1361)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1225)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ArrayIndexOutOfBoundsException: 2
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1359)
	... 17 more
]]>
    </failure>
  </testcase> <!-- registerPage -->
  <testcase name="findRegisterElement" time="0.029" classname="net.groot.data.TestNgRegisterData"/>
</testsuite> <!-- net.groot.data.TestNgRegisterData -->
